mtcnn-0001-merge-pr-129.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/setup.py b/setup.py
```

Comment:
shall we move this to a folder instead of a root? otherwise it will make project messy.